ios/ConvosNSE/SentryManager.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+import Sentry
+
+final class SentryManager {
+  static let shared = SentryManager()
+
+  private var sentryInitialized = false
+
+  private init() {}
+
+  func initSentry() {
+    DispatchQueue.main.async {
+      guard !self.sentryInitialized else { return }
+
+      let sentryEnv = try! getInfoPlistValue(key: "Env", defaultValue: "dev")
```

Comment:
@jarodl I think we don't have any value for "Env" right? I don't mind adding it to the info.plist. 

but right now we just have "XmtpEnvironment". Maybe i can replace "XmtpEnvironment" with just "Env" and we use this variable everywhere to determine the Env